blocks/career-carousel/career-carousel.js

```diff
@@ -0,0 +1,158 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { fetchIndex } from '../../scripts/scripts.js';
+
+export function filterIncompleteEntries(json) {
+  return json.data.filter((e) => e.image !== '' && e['career-quote'] !== '0' && e['career-jobtitle'] !== '0');
+}
+
+export function scrollToCard(idx, card, precedingCard, slides, span, doc) {
+  const rect = card.getBoundingClientRect();
+
+  // set the style on the active button
+  const buttons = doc.querySelectorAll('.career-slides-nav span.active-nav');
+  buttons.forEach((b) => b.classList.remove('active-nav'));
+  span.classList.add('active-nav');
+
+  // Compute the gap to add to the location
+  let gap = 0;
+  if (precedingCard) {
+    const prevRect = precedingCard.getBoundingClientRect();
+    gap = rect.x - (prevRect.x + prevRect.width);
+  }
+
+  slides.scrollTo(idx * (rect.width + gap), slides.scrollHeight);
+}
+
+export function scrollToAdjacent(spans, slideDivs, slides, next, doc) {
+  let curActive;
+  for (let i = 0; i < spans.length; i += 1) {
+    if (spans[i].classList.contains('active-nav')) {
+      curActive = i;
+      break;
+    }
+  }
+
+  if (curActive === undefined) {
+    return;
+  }
+
+  let newActive = curActive + (next ? 1 : -1);
+  if (newActive < 0) {
+    newActive = spans.length - 1;
+  } else if (newActive >= spans.length) {
+    newActive = 0;
+  }
+
+  if (slideDivs.length <= newActive) {
+    return;
+  }
+
+  scrollToCard(
+    newActive,
+    slideDivs[newActive],
+    newActive > 0 ? slideDivs[newActive - 1] : null,
+    slides,
+    spans[newActive],
+    doc,
+  );
+}
+
+export default async function decorate(block) {
+  const json = await fetchIndex('query-index', 'career-testimonials');
+  const data = filterIncompleteEntries(json);
+
+  const careerSlider = document.createElement('div');
+  careerSlider.classList.add('career-slider');
+
+  const careerSlides = document.createElement('div');
+  careerSlides.classList.add('career-slides');
+
+  const slideDivs = [];
+  for (let i = 0; i < data.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('career-card');
+    const a = document.createElement('a');
+    a.href = data[i].path;
+    const pic = createOptimizedPicture(data[i].image, data[i].pagename);
+    a.append(pic);
+
+    const bqc = document.createElement('div');
+    bqc.classList.add('career-card-bqc');
+    const bq = document.createElement('blockquote');
+    bq.textContent = data[i]['career-quote'];
+    bqc.append(bq);
+    a.append(bqc);
+
+    const nm = document.createElement('h6');
+    nm.textContent = data[i].pagename;
+    a.append(nm);
+
+    const role = document.createElement('p');
+    role.textContent = data[i]['career-jobtitle'];
+    a.append(role);
+
+    const link = document.createElement('button');
+    link.textContent = 'Read More '; // TODO
+    const arrow = document.createElement('img');
+    arrow.src = '/icons/angle-right-blue.svg';
```

Comment:
Why its needed here, can it be in css

Reply:
I guess it can be but then it has to be changed to a background image, these arrows arent really background images...